intg/src/main/java/org/apache/atlas/model/discovery/AtlasSearchResult.java

```diff
@@ -58,6 +59,10 @@ public class AtlasSearchResult implements Serializable {
     private Map<String, Object>            aggregations;
     private Map<String,Double>             searchScore;
 
+    private Map<String, SearchMetadata>   searchMetadata;
```

Comment:
This is becoming a little confusing, SearchMetadata is Map of SearchMetadata. We can do better naming here.

Reply:
refactored